pkg/agent/openflow/pipeline_test.go

```diff
@@ -309,6 +262,113 @@ func TestBuildPipeline(t *testing.T) {
 	}
 }
 
+func prepareMulticastPipeline() testPipeLine {
```

Comment:
I think you should add unit test for function `InstallMulticastInitialFlows` instead of `IGMPEgressFlow`, and use gomock to verify your expected APIs in FlowBuilder is called when the flag `enablePolicy` is true.

Reply:
ok